doc/qbk/5.1.CharSet.qbk

```diff
@@ -13,7 +13,7 @@ A ['CharSet] is a unary predicate which is invocable with
 this equivalent signature:
 
 ```
-bool( char ch ) const;
+bool( char ch ) const noexcept;
```

Comment:
But you need to go through all the charsets provided by the library (e.g. `lut_chars`) and make sure they are noexcept. And you should make sure that the documentation does not imply the `constexpr` requirement on `bool operator()( char )`. In fact we should remove the `constexpr` designation from our signatures of that function.

Reply:
Yes. I did check that. The examplar in snippets.cpp has no constexpr. I'll check the other existing functions.